spec/teacher_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../teacher'
+
+describe Teacher do
+  before :each do
+    @teacher = Teacher.new 15, 'Chemistry', 'Teacher', 'Ahmed'
+  end
+
+  describe '#new' do
+    it 'takes four parameters and returns a Teacher object' do
+      @teacher.should be_an_instance_of Teacher
+    end
+  end
+  describe '#name' do
+    it 'returns whether teacher can use service' do
+      @teacher.can_use_services?.should be_truthy
+    end
```

Comment:
- [ Required ] I  think it would be better if you add a test case for the  `can_use_services` method in the `Teacher` class. According to the project requirements, all the methods in the `Teacher` class should be tested. Please see the example below as a suggestion on how to achieve this.

## Example code
```ruby
    it "The can_use_services? method returns true if over age" do
      person = Teacher.new(19, "Mathematics", "teacher", "Sam")
      expect(person.can_use_services?).to be true
    end
```

Reply:
Hi @Felix45. Thank you for your feedback, but I don't understand what the problem is in this test specifically. I can see that it's working fine and performing exactly what is expected in your example